services/appflowy-collaborate/src/group/broadcast.rs

```diff
@@ -314,7 +317,7 @@ async fn handle_client_messages<Sink>(
         Ok(response) => {
           trace!("[realtime]: sending response: {}", response);
           match sink.send(response.into()).await {
-            Ok(_) => {},
+            Ok(()) => {},
```

Comment:
what is the difference using `()` instead of `_`

Reply:
i prefer `()` as it gives move information(that the returned value is nothing, so we dont have to anything), as compared to `_` which indicate that the returned value can be something, but we choose to ignore it